docs/understand_the_basics.md

```diff
@@ -14,3 +14,74 @@ Before effectively using Argo CD, it is necessary to understand the underlying t
 * If you're integrating with a CI tool:
 	* [GitHub Actions Documentation](https://docs.github.com/en/actions)
 	* [Jenkins User Guide](https://www.jenkins.io/doc/book/)
+
+# Argo CD Contribution Guide
+
+## Explore the Project Structure
+
+To get started with contributing, it's helpful to familiarize yourself with the project structure. Some key folders/files to check:
+- **`cmd/`**: Contains the main entry points for the various Argo CD commands/tools.
+- **`pkg/`**: Contains reusable packages used throughout the project.
+- **`manifests/`**: Kubernetes manifests for deploying Argo CD.
+- **`docs/`**: Documentation files explaining various components.
+- **`README.md`**: Overview of the project, including setup and usage instructions.
```

Comment:
I'd rather skip this part of improve it. In future, the project structure _may_ change too. Reading the documenation that is written, i think we can add more to it else remove it entirely. Rn, it looks vague and GPTed stuff.